k8s/gob/web/rc.yml

```diff
@@ -0,0 +1,53 @@
+---
+kind: ReplicationController
+apiVersion: v1
+metadata:
+  namespace: default
+  name: web
+
+spec:
+  replicas: 1
+  selector:
+    app: web
+  template:
+    metadata:
+      labels:
+        app: web
+
+    spec:
+      dnsPolicy: ClusterFirst
+      volumes:
+        - name: linkerd-config
+          configMap:
+            name: linkerd-config
+
+      containers:
+        - name: svc
+          image: gobsvc/web:v0
+          args:
+            - --srv=:8080
+            - --gen-addr=localhost:4180
+            - --word-addr=localhost:4180
```

Comment:
It's not technically necessary, but I think it's visually easier to parse this file if you quote these flags. Also, that would be consistent with how you've set it up in the other to rc files.

    - "--srv=:8080"
    - "--gen-addr=localhost:4180"
    - "--word-addr=localhost:4180"